compiler/cpp/src/thrift/generate/t_rs_generator.cc

```diff
@@ -1406,7 +1411,6 @@ void t_rs_generator::render_struct_sync_write(
 ) {
   f_gen_
     << indent()
-    << visibility_qualifier(struct_type)
```

Comment:
I assume the reason why this can be removed is because the methods on the trait are already in the public scope?

Reply:
Also, please note that as a result of this _all_ structs (args and results included) suddenly have public read/write methods. This was not intended originally, and it's not clear we want it as a result of this PR as well.